City-of-Helsinki/berth-reservations
# VEN-969 | Load emails to DB

## Description :sparkles:
* Add a script to load the templates to the database
* Set an enviornemnt variable on the deployments that runs the command

## Issues :bug:
### Closes :no_good_woman:
**[VEN-969](https://helsinkisolutionoffice.atlassian.net/browse/VEN-969):** Add berth email templates to the database

### Related :handshake:
#376 
#383 

## Testing :alembic:
### Manual testing :construction_worker_man:
1. Login to the review environment:
https://berth-reservations-qa-r-ven-969-add-sb6qn9.test.kuva.hel.ninja
```
user: admin
pass: adminpass
```
2. Go to the `Renew berth order` notification template
https://berth-reservations-qa-r-ven-969-add-sb6qn9.test.kuva.hel.ninja/admin/django_ilmoitin/notificationtemplate/5/change/

3. You'll see both HTML and plain text emails
4. Hit `Preview` at the bottom of the page, it will open a pop-up with the rendered HTML preview
5. Check the rest of the templates

## Screenshots 📸 
The plain text template is not that straightforward to preview, but manually rendering it (will be done automatically when the email is sent), this is what it looks like:
<img width="1263" alt="image" src="https://user-images.githubusercontent.com/15201480/101025480-513c7480-357e-11eb-8652-adce9c8f8cb4.png">
